src/librustdoc/clean/mod.rs

```diff
@@ -1505,7 +1505,11 @@ impl Clean<Type> for hir::Ty<'_> {
 /// Returns `None` if the type could not be normalized
 #[allow(unreachable_code, unused_variables)]
```

Comment:
```suggestion
```
Making the return conditional should make everything reachable here.

Reply:
Thanks, fixed!